src/api/getLanglinks.js

```diff
@@ -16,34 +21,37 @@ export const getLanglinks = (lang, title) => {
   return cachedFetch(url, response => {
     const { pages } = response.query
     const langlinks = pages[0].langlinks
-    const prioritizedLanguages = langlinks
-      .filter(item => {
-        return prioritizedList.indexOf(item.lang) !== -1 && isSupportedForReading(item.lang)
-      })
+    const allLanguages = langlinks
       .map(item => {
         return {
           title: item.autonym,
           langname: item.langname,
-          lang: item.lang,
+          lang: convertToWikiLang(item.lang),
           description: item.title,
           dir: getDirection(item.lang)
         }
       })
-    const allLanguages = prioritizedLanguages.concat(
-      langlinks
-        .filter(item => {
-          return isSupportedForReading(item.lang) && prioritizedList.indexOf(item.lang) === -1
-        })
-        .map(item => {
-          return {
-            title: item.autonym,
-            langname: item.langname,
-            lang: item.lang,
-            description: item.title,
-            dir: getDirection(item.lang)
-          }
-        })
-    )
+      .filter(item => isSupportedForReading(item.lang))
```

Comment:
`filter` should go first then `map`, so unsupported language don't need to go through `map`

Reply:
I prefer to map first so we filter with the converted land code.